api/campaigns/class-campaign-data-utils.php

```diff
@@ -340,4 +368,18 @@ public static function is_inline( $popup ) {
 	public static function is_above_header( $popup ) {
 		return isset( $popup->t ) && 'a' === $popup->t;
 	}
+
+	/**
+	 * Get all events types.
+	 */
+	public static function get_all_events_types() {
```

Comment:
"all event types" isn't entirely accurate here, as there are also `prompt_seen` and `prompt_dismissed` types. Maybe `get_all_reader_events_types`?

Reply:
Refactored to `get_reader_events_types` in f379d0b803eefd128911c0bcbb533ec819427320